src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 17px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-1n + 2),
```

Comment:
The `:nth-child(-1n + 2)` selector is incorrect. It should be `:nth-child(-n + 2)` to select the first two children.